extensions/frontend-tiles/src/integration-test/IntegrationTest.test.ts

```diff
@@ -0,0 +1,99 @@
+import { TileAdmin } from "@itwin/core-frontend";
+import { expect } from "chai";
+import { TestContext } from "./TestContext";
+import { GraphicRepresentationStatus } from "../GraphicsProvider/GraphicRepresentationProvider";
+
+async function getExport(exportId: string, accessToken: string, urlPrefix?: string): Promise<any> {
+  const headers = {
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    Authorization: accessToken,
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+  };
+
+  // obtain export for specified export id
+  const url = `https://${urlPrefix}api.bentley.com/mesh-export/${exportId}`;
+  try {
+    const response = await fetch(url, { headers });
+    return await response.json();
+  } catch (err) {
+    throw new Error(`Failed to get export: ${err}`);
+  }
+}
+
+describe("MeshExportServiceIntegrationTest", () => {
+  let accessToken: any;
+  let urlPrefix: any;
+  let exportId: any;
+
+  before(async () => {
+
+    // obtain access token
+    const testContext = await TestContext.instance();
+    accessToken = testContext.getAccessToken();
+
+    const iModelId = process.env.MES_IMODEL_ID || "";
+    urlPrefix = process.env.IMJS_URL_PREFIX || "";
+
+    const requestOptions: RequestInit = {
+      method: "POST",
+      headers: {
+        /* eslint-disable-next-line @typescript-eslint/naming-convention */
+        "Authorization": accessToken,
+        /* eslint-disable-next-line @typescript-eslint/naming-convention */
+        "Accept": "application/vnd.bentley.itwin-platform.v1+json",
+         
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        iModelId,
+        changesetId: "",
+        exportType:"IMODEL",
+      }),
+    };
+
+    // initiate mesh export
+    const response = await fetch(`https://${urlPrefix}api.bentley.com/mesh-export/`, requestOptions);
+    const result = await response.json();
```

Comment:
shouldnt you check the response was ok first?

Reply:
check has been added in latest commit